lib/comply/version.rb

```diff
@@ -1,3 +1,3 @@
 module Comply
-  VERSION = '1.8.0'
+  VERSION = '1.8.1'
```

Comment:
This is a larger change than I had originally anticipated when we discussed it; feels more like 1.9.0.